tests/e2e/controlplane/control_plane_test.go

```diff
@@ -203,7 +203,7 @@ spec:
 
 				When("sample pod is deployed", func() {
 					BeforeAll(func() {
-						if isAlias(name, version) {
+						if isAlias(version.Name, version) {
```

Comment:
As far as I can tell, `version` can never be an alias, because it comes from `istioversion.GetLastestPatchVersions()`, which gets the versions from `istioversions.List`, which only contains actual versions (no aliases).

Also, this function call seems very wrong. It passes in `version.name` and `version`, and the `isAlias` function then returns `name != version.Name`, so effectively it's returning `version.Name != version.Name`, which is always false. 

I suggest we remove `isAlias`.